docs/v3/resources/upgrade-agents-to-workers.mdx

```diff
@@ -7,7 +7,7 @@ Upgrading from agents to workers significantly enhances the experience of deploy
 by simplifying the specification of each flow's infrastructure and runtime environment.
 
 <Note>
-This guide is for users who are on `prefect<3.0` who are upgrading from agents to workers.
+This guide is for users who are upgrading from agents (a deployment pattern specific to `prefect>2.0,<3.0`) to workers.
```

Comment:
you may either be upgrading from agents to workers while staying on 2.x or upgrading to workers because you want to move to 3.x